seccomp/src/lib.rs

```diff
@@ -1198,7 +1756,68 @@ mod tests {
             .collect(),
             SeccompAction::Trap,
         )
-        .unwrap();
+        .unwrap()
+    }
+
+    #[test]
+    fn test_filter_bpf_output_dword() {
+        // Compares translated filter with hardcoded BPF program.
+        {
+            let mut empty_rule_map = BTreeMap::new();
+            empty_rule_map.insert(1, vec![]);
+            assert!(SeccompFilter::new(empty_rule_map, SeccompAction::Allow).is_err());
+        }
+
+        let filter = create_test_bpf_filter(ArgLen::DWORD);
+
+        let instructions = vec![
```

Comment:
How did you obtain the following hardcoded instructions? Please add a comment with the steps so people that want to change this in the future know how to.

Reply:
I did it manually. I started from the instructions for `test_filter_bpf_output_qword` which were already present and modified them accordingly. I don't think it's worth a comment.

Reply:
You simply need to learn the voodoo incantation 😆

Reply:
Ok. Hope you'll modify them the next time cause I ain't got any idea how to do that.